commands/lan.go

```diff
@@ -204,6 +204,7 @@ func RunLanList(c *core.CommandConfig) error {
 }
 
 func RunLanGet(c *core.CommandConfig) error {
+	c.Printer.Infof("Lan with id: %v is getting...", viper.GetString(core.GetFlagName(c.NS, config.ArgLanId)))
```

Comment:
Maybe also the datacenter id is good to add? This applies to all resources created in a datacenter/depending on other resources.

Reply:
done